core/templates/components/ck-editor-helpers/ck-editor-4-widgets.initializer.ts

```diff
@@ -25,11 +25,16 @@ require('services/context.service.ts');
 
 angular.module('oppia').run([
   '$compile', '$rootScope', '$timeout', 'RteHelperService',
-  'HtmlEscaperService', 'ContextService',
+  'HtmlEscaperService', 'ContextService', 'ENABLE_LITERALLYCANVAS_EDITOR',
   function($compile, $rootScope, $timeout, RteHelperService,
-      HtmlEscaperService, ContextService) {
+      HtmlEscaperService, ContextService, ENABLE_LITERALLYCANVAS_EDITOR) {
     var _RICH_TEXT_COMPONENTS = RteHelperService.getRichTextComponents();
     _RICH_TEXT_COMPONENTS.forEach(function(componentDefn) {
+      if (componentDefn.id === 'svgeditor') {
+        if (!ENABLE_LITERALLYCANVAS_EDITOR) {
+          return;
+        }
+      }
```

Comment:
Add TODO here for the removal of this after everything is fully implemented.

Reply:
Done.